servers/rendering/renderer_canvas_cull.cpp

```diff
@@ -1923,6 +1923,12 @@ void RendererCanvasCull::canvas_light_occluder_set_light_mask(RID p_occluder, in
 
 	occluder->light_mask = p_mask;
 }
+void RendererCanvasCull::canvas_light_occluder_set_z_index(RID p_occluder, int z_index) {
```

Comment:
```suggestion

void RendererCanvasCull::canvas_light_occluder_set_z_index(RID p_occluder, int p_z_index) {
```
Leave a line between methods